cl/blockbuilder/blockbuilder.go

```diff
@@ -476,6 +476,10 @@ func (bb *BlockBuilder) pushNewPayload(ctx context.Context, executionPayload eng
 	emptyVersionHashes := []common.Hash{}
 	return retryFunc(func() error {
 		status, err := bb.engineCl.NewPayloadV4(ctx, executionPayload, emptyVersionHashes, &hash, []hexutil.Bytes{})
+		bb.logger.Debug("newPayload result",
+			"status", status.Status,
+			"validationError", status.ValidationError,
+			"latestValidHash", status.LatestValidHash)
```

Comment:
nit: Indentation in this case should ideally be
```suggestion
			"latestValidHash", status.LatestValidHash,
		)
```

Improves clarity with line between next control statement